includes/Modules/Analytics_4/Settings.php

```diff
@@ -178,51 +170,4 @@ function( $dimension ) {
 		};
 	}
 
-	/**
```

Comment:
This is outdated in favour of migration script done in #8082, removed also in this [commit](https://github.com/google/site-kit-wp/pull/8123/commits/277fcddd2d2d17e5145c8888f42eb2eda11b6755)